### ISSUE TITLE
when use STRATEGY_FILL_VIEW, gravity center invalid

### ISSUE THREAD

**Issue by ColorfulHorse** [BelooS/ChipsLayoutManager#64]
itemView gravity center invalid, it seems cause by layout view without measure

**Comment by martin-braun** [comment]
Yes, I can confirm that. Items are left aligned in their container that will be stretched on STRATEGY_FILL_VIEW. A possible bypass could be to fill the contents to the container, but even setting `fill_parent` on the `layout_width` will be ignored. Last commit 4 years ago, dead.

**Comment by ColorfulHorse** [comment]
> Yes, I can confirm that. Items are left aligned in their container that will be stretched on STRATEGY_FILL_VIEW. A possible bypass could be to fill the contents to the container, but even setting `fill_parent` on the `layout_width` will be ignored. Last commit 4 years ago, dead.

flexbox instead

**Comment by martin-braun** [comment]
@ColorfulHorse Flexbox was not right for me, I used the GridLayoutManager, instead.
